packages/block-library/src/paragraph/edit.js

```diff
@@ -64,12 +64,26 @@ function ParagraphBlock( {
 	const blockProps = useBlockProps( {
 		ref: useOnEnter( { clientId, content } ),
 		className: classnames( {
-			'has-drop-cap': dropCap,
+			'has-drop-cap':
+				align === ( isRTL() ? 'left' : 'right' ) || align === 'center'
```

Comment:
Minor thing - not a blocker. Would it be more readable if we wrote it like this?
```suggestion
				align === center || ( isRTL() && align === 'left' ) || ( ! isRTL() && align === 'right' )
```

Reply:
Personally this is not easier for me to read.